### ISSUE TITLE
Step 1 keyword search usability

### ISSUE THREAD

**Issue by ggalibert** [aodn/aodn-portal#2488]
This is an issue that has been witnessed by me, Seb and Bene during the last TAMS practical, where the IMOS and the AODN portal were presented to 20 chinese students. At least 5 of them faced the same usability problem.

**What happens**
Looking for the hawksbill sea turtle data collection, I start typing "hawksbill sea turtle" in the keyword search and then I'm facing 3 problems: 
1. I don't have any hint telling me that hitting Enter would trigger the search.
2. I'm guessing I must click the on the magnifier logo. Unfortunately, that didn't work. 
3. Ok I must have missed something in the keyword search field so I go back in there but when doing so, I loose the text I had typed!

**What should happen**
Several improvements can be suggested for each problem mentioned above:
1. Change the initial background grey text of the keyword search field from "Keyword Search..." to "Keyword search and press Enter...". Or add a button Enter. Or add a tooltip somewhere.
2. Trigger the search when one clicks on the magnifier logo.
3. Do not clear the typed text when I leave the field and go back to it for editing/tuning the search for example.

**Comment by pblain** [comment]
We've been running a usability experiment on this using Google Analytics since the 5th of May. If the TAMS practical was on or after the 5th of May it's included in these results. The results (thus far and based on 358 events) are as follows:
* 78% of users who go to the keyword search click enter. We can assume these guys are not confused by lack of a button.
* 3% click the magnifying glass. We can assume these guys are confused - at least initially.
* 19% go to the keyword search and neither click enter or the magnifying glass. Unfortunately we can't measure how many of these don't realise that they should click enter, and how many go to the keyword search but just decide not to type or search for anything. We can assume, I think, that at least some of this 19% are confused.

In conclusion, it seems that somewhere between 3% and 22% of users are confused (at least initially) by the lack of a button on the keyword search. If we split it down the middle and say that 14% are confused it may be worth reintroducing the button.

**Comment by ggalibert** [comment]
TAMS workshop was held the 2nd of May and wasn't captured unfortunately.

Note in the suggested improvements that there are lot less time consuming tasks than re-introducing a button, in particular, the keyword search fied background text in grey.

**Comment by ggalibert** [comment]
Following [line](https://github.com/aodn/aodn-portal/blob/45eebfe7d60d4eb8c881e1d91b6b43d188223260/web-app/js/portal/lang/en.js#L173) could be updated to: "Type keyword(s) and hit Enter.."

**Comment by ggalibert** [comment]
See PR https://github.com/aodn/aodn-portal/pull/2489

**Comment by pblain** [comment]
Update: As of today we have 894 events in Google Analytics. Unfortunately we missed a week's worth of stats, but as soon as we hit 1000 we'll test the proposed change and compare the results.

**Comment by ggalibert** [comment]
Can you track events from new users (first timers on the portal)? This is the kind of event that would be relevant.

**Comment by pblain** [comment]
@ggalibert - I'll have to look into it, but GA does differentiate new users from return users, so I suspect that it's possible.

**Comment by ggalibert** [comment]
Ok, the difficulty of this exercise is that a new user doesn't go straight for a text search, or a return user might try it for the first time later. 

I don't know if that's possible to identify the first time a user is using the text search.

**Comment by pblain** [comment]
@ggalibert - You're right. However, we do know that all new users who use the search have never used the search before.

**Comment by pblain** [comment]
At the time of writing we have 1,117 events in GA where users moved focus to the keyword search. The results are:
### All Users (1,117 events)
76% hit enter
4% clicked the magnifying glass
20% did something else
### New Users Only (427 events)
81% hit enter
2% clicked the magnifying glass
17% did something else

**Comment by pblain** [comment]
## Acceptance Criteria (B Test)
1. The default grey text reads: "Type keyword, press enter".
1. The text entered by the user is not cleared when re-entering the text box.
1. The default grey text is cleared when the user enters the text box.
1. The default grey text is re-instantiated if the box is cleared or left empty by the user.
1. Tool tip added that reads: "Type a keyword and press enter to narrow search results".

Note: After we compare the current A-test results with this B-test we will test the impact of a "'Go" button.

**Comment by pblain** [comment]
## A/B Test Final Results
Test A measures user response to Phil's original design.
Test B measures user response to GG's suggestions.

New users are perhaps the most interesting to look at because we know that they have not seen the UI before.

## Test A - Phil's original design
### All Users (1,117 events)
76% hit enter
4% clicked the magnifying glass
20% did something else
### New Users Only (427 events)
81% hit enter
2% clicked the magnifying glass
17% did something else

## Test B - GG's suggestions
### All Users (1,594 events)
75% hit enter
3% clicked the magnifying glass
22% did something else
### New Users Only (594 events)
78% hit enter
2% clicked the magnifying glass
20% did something else

## Conclusion
GG's suggestions sounded sensible at the time, but it appears that the changes made no significant difference to the proportion of users who erroneously click on the magnifying glass. Only about 2% of users made this mistake in both A and B tests. 

There was also no significant change in the proportion of users who go to the keyword search but then fail to either hit enter or click the magnifying glass. About 20% in both cases. We don't know exactly what these users are doing, but we do know that the proportion of users behaving this way remained the same in both tests.

Given that both designs appear to be equivalent in user response I suggest that we leave it how it is.
